ballerina/Ballerina.toml

```diff
@@ -3,7 +3,7 @@ org = "ballerinax"
 name = "ibm.ibmmq"
 version = "0.1.4"
 authors = ["Ballerina"]
-keywords = ["ibm-mq"]
+keywords = ["ibm.ibmmq"]
```

Comment:
Shall we include a few more related keywords in here?

Reply:
Fixed with [bf8d879701fcfdd5eac7367a7b1d1299a9899ffd](https://github.com/ballerina-platform/module-ballerinax-ibm.ibmmq/pull/42/commits/bf8d879701fcfdd5eac7367a7b1d1299a9899ffd)